x/tokenfactory/keeper/grpc_query.go

```diff
@@ -11,7 +11,7 @@ import (
 
 var _ types.QueryServer = Keeper{}
 
-func (k Keeper) Params(ctx context.Context, _ *types.QueryParamsRequest) (*types.QueryParamsResponse, error) {
+func (k Keeper) Params(ctx context.Context, req *types.QueryParamsRequest) (*types.QueryParamsResponse, error) {
```

Comment:
`req` is not used anywhere

Reply:
This is the same as in original `osmosis` module